recipes/libtiff/all/patches/4.5.0-0001-cmake-dependencies.patch

```diff
@@ -0,0 +1,116 @@
+--- a/cmake/DeflateCodec.cmake
++++ b/cmake/DeflateCodec.cmake
+@@ -35,7 +35,10 @@ set(LIBDEFLATE_SUPPORT FALSE)
+ 
+ # libdeflate
+ set(LIBDEFLATE_SUPPORT FALSE)
+-find_package(Deflate)
++find_package(libdeflate CONFIG)
```

Comment:
With CMakeDeps its possible for consumer to override the generated names to match